Modules/_queuemodule.c

```diff
@@ -358,42 +359,51 @@ PyDoc_STRVAR(queue_module_doc,
 This module is an implementation detail, please do not use it directly.");
 
 static struct PyModuleDef queuemodule = {
-    PyModuleDef_HEAD_INIT,
-    "_queue",
-    queue_module_doc,
-    -1,
-    NULL,
-    NULL,
-    NULL,
-    NULL,
-    NULL
+    .m_base = PyModuleDef_HEAD_INIT,
+    .m_name = "_queue",
+    .m_doc = queue_module_doc,
+    .m_size = sizeof(simplequeue_state),
```

Comment:
Shouldn't you define `m_clear` and `m_free` to clear the `EmptyError` and `SimpleQueueType` members of the state?

Reply:
Oops, of course! Comin' up.